app/src/main/java/org/oppia/app/topic/revision/TopicRevisionFragmentPresenter.kt

```diff
@@ -89,13 +88,4 @@ class TopicRevisionFragmentPresenter @Inject constructor(
         setViewModel = TopicRevisionSummaryViewBinding::setViewModel
       ).build()
   }
```

Comment:
Not able to comment on that particular line but writing here. 
There is an unused import at line number 13 `import org.oppia.app.model.EventLog`

Reply:
Done.